src/components/application_manager/src/commands/mobile/subscribe_vehicle_data_request.cc

```diff
@@ -250,6 +250,15 @@ void SubscribeVehicleDataRequest::on_event(const event_engine::Event& event) {
       result_code = mobile_apis::Result::IGNORED;
       response_info = "Already subscribed on some provided VehicleData.";
     }
+
+    if (!vi_waiting_for_subscribe_.empty()) {
```

Comment:
@AKalinich-Luxoft 
Shouldn't this container to be cleaned off succeeded subscriptions?